css/styles.css

```diff
@@ -233,6 +233,11 @@ a {
 
     .input-search {
         height: 50px;
+        padding-right: 50px;
```

Comment:
Instead of padding here, let’s adjust it here so we’re consistent with the left side padding:

```diff
diff --git a/css/styles.css b/css/styles.css
index 9cac9e2..f6679a1 100644
--- a/css/styles.css
+++ b/css/styles.css
@@ -108,7 +108,7 @@ a {
 #v {
     font-size: 16px;
     margin: 0 auto;
-    padding-left: 50px;
+    padding-left: 0 50px;
     text-align: center;
     width: 100%;
 }

```

Reply:
This raises a broader question about overall cleaning up CSS selectors and not using IDs, but that’s a much bigger code change that’s just a “nice to have” one day.

Reply:
@mukundsutar hi, any chance you can revisit this?